readthedocs.yml

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
This space at the beginning is not required, correct?

Reply:
I don't know why I do that but I'm happy enough without it :)